.tool-versions

```diff
@@ -1,4 +1,3 @@
-nodejs 16.14.0
+nodejs 16.17.0
 elixir 1.13.3
 erlang 24.3.4
-ruby 2.6.6
```

Comment:
We still use `foreman` to run the `bin/setup´ script. While it's true that any version of Ruby should work for this, I'd like to have the Ruby dependency explicit. What do you think?

Reply:
Sure no problem. It's probably faster to install `foreman` from `brew` or `apt-get` though. (ruby build takes ages)

Reply:
Yeah, I agree. We should probably find a different solution for using `foreman` just to start the local development environment. It's very convenient but might be a pain for the first setup of the repo.

For now, let's keep this and I'll come up with something on a later PR